safedelete/tests/test_soft_delete.py

```diff
@@ -104,3 +104,28 @@ def test_validate_unique(self):
                 name='test'
             ).validate_unique
         )
+
+    def test_check_unique_fields_exists(self):
+        # No unique fields
+        self.assertEqual(self.instance.has_unique_fields(self.instance), False)
```

Comment:
Use the classes and not the instances to call class methods or static methods.

Reply:
done.